Flutter/pdf/working-with-headers-and-footers.md

```diff
@@ -107,3 +107,7 @@ File('HeaderAndFooter.pdf').writeAsBytes(document.save());
 document.dispose();
 
 {% endhighlight %}
+
+By executing the above code sample, you will get the PDF document as follows.
+
+![Header and Footer PDF](images/working-with-headers-and-footers/header-and-footer.png)
```

Comment:
Hi @AnanthaGokulaRaman ,

The provided output image and the example code for the header and footer are wrong. The header content must be present before the page content starts. 

So update the code like draw the page content once assigned the header and footer template.